app/src/main/kotlin/co/touchlab/droidconandroid/VoteDetailFragment.kt

```diff
@@ -0,0 +1,105 @@
+package co.touchlab.droidconandroid
+
+import android.os.Bundle
+import android.support.v4.app.DialogFragment
+import android.view
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.RatingBar
+import android.widget.TextView
+import co.touchlab.android.threading.eventbus.EventBusExt
+import co.touchlab.droidconandroid.data.DatabaseHelper
+import co.touchlab.droidconandroid.data.TalkSubmission
+import co.touchlab.droidconandroid.tasks.persisted.PersistedTaskQueueFactory
+import co.touchlab.droidconandroid.tasks.persisted.UpdateVotePersisted
+import co.touchlab.droidconandroid.ui.RemoveTalkListener
+
+
+/**
+ *
+ * Created by toidiu on 8/5/15.
+ */
+class VoteDetailFragment : DialogFragment() {
+
+    companion object {
+        val TALK = "TALK"
+
+        fun newInstance(talk: TalkSubmission): VoteDetailFragment {
+            val fragment = VoteDetailFragment()
+            val args = Bundle()
+            args.putSerializable(TALK, talk)
+            fragment.arguments = args
+            fragment.isCancelable = true
+            return fragment
+        }
+    }
+
+    var rating: RatingBar? = null
+    var talk: TalkSubmission? = null
+
+    override fun onCreateView(inflater: LayoutInflater?, container: view.ViewGroup?, savedInstanceState: Bundle?): View? {
+        return inflater!!.inflate(R.layout.fragment_vote_detail, null)
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super<DialogFragment>.onCreate(savedInstanceState)
+        talk = arguments.getSerializable(TALK) as TalkSubmission
+
+    }
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super<DialogFragment>.onActivityCreated(savedInstanceState)
+
+        initEvent()
```

Comment:
The view is created when you inflate it in onCreateView. after which you can safely initialize it. So in the rare cases where the fragment is destroyed but not the activity, it would be safer there. But certainly an edge case